docs/contact.html

```diff
@@ -0,0 +1,99 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="index.html" >
+    <script src="script.js"></script>
+    <link rel="stylesheet" type="text/css" href="styles/style.css" />
+    
+    <title>Contact</title>
+</head>
+<body>
+<div class="contcol"> <!--for the entire page-->
+    <p class="dayclock" id="datetime"></p>
+    <div class="headher">
+    <h1> Contact Me!! </h1>
+ </div>   
+
+ <div class="headher">
+ <h2>
+    <a href="index.html">  Home </a>
+</h2>
+</div>
+
+<div>
+    <caption>
+        Leave me a message:
+    </caption>
```

Comment:
FYI, `<caption>` is intended for use inside a `<table>` element: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/caption